src/main/java/game/racing/RacingGame.java

```diff
@@ -10,14 +10,14 @@
  * @author : yusik
  * @date : 2019/10/26
  */
-public class RacingGame implements Game {
+public class RacingGame implements Game<RacingResult> {
 
-    private int numberOfCar;
+    private String[] carNames;
```

Comment:
이왕 만든 `Car`를 사용하면 어떨까요?

```suggestion
    private List<Car> cars;
```